benches/benches/prelude.rs

```diff
@@ -42,6 +43,55 @@ pub fn bench_ternary<const BITS: usize, const LIMBS: usize, U>(
     );
 }
 
+pub fn bench_shifting_binop<const BITS: usize, const LIMBS: usize, U>(
```

Comment:
why can you not just call `bench_arbitrary_with(..., (U256::arbitrary(), 0..=BITS))`? none of these look necessary or useful

Reply:
tbh did not realize I could do that